cpp/src/neighbors/brute_force.cu

```diff
@@ -15,8 +15,11 @@
  */
 
 #include "./detail/knn_brute_force.cuh"
+
+#include <cuvs/core/bitmap.hpp>
 #include <cuvs/neighbors/brute_force.hpp>
 
+#include <raft/core/bitset.cuh>
```

Comment:
Hi @cjnolet @benfred @lowener , I can't remove this inclusion, because it will cause a compilation issue of `n_elements` of `bitset` reference but redefinition.  I appreciate if you have any idea better to process. Thank you guys!

Reply:
The include  of the bitmap.cuh should be where `n_elements` is called, in `cpp/src/neighbors/detail/knn_brute_force.cuh`.